bit/wallet.py

```diff
@@ -813,6 +835,9 @@ def prepare_transaction(
         :param unspents: The UTXOs to use as the inputs. By default Bit will
                          communicate with the blockchain itself.
         :type unspents: ``list`` of :class:`~bit.network.meta.Unspent`
+        :type unspents: ``list`` of :class:`~bit.network.meta.Unspent`
```

Comment:
Line appears twice.

Reply:
fixed in commit 7b4d000cb3d318d5d47274f3c6f5f0d054696868